build.sh

```diff
@@ -209,7 +209,9 @@ cd skia
 /bin/rm -rf src/c include/c
 cp ../../capi/sk_capi.h include/
 cp ../../capi/sk_capi.cpp src/
-grep -v "/c/" gn/core.gni | grep -v "sk_capi.cpp" | sed -e 's@skia_core_sources = \[@&\n  "$_src/sk_capi.cpp",@' >gn/core.revised.gni
+
+grep -v "/c/" gn/core.gni | grep -v "sk_capi.cpp" | sed -e 's@skia_core_sources = \[@&\
+  "$_src/sk_capi.cpp",@' >gn/core.revised.gni
```

Comment:
I'm surprised this was necessary for you. I developed this on macOS originally.

Reply:
the original code simply adds 'n' character to the output file (breaking the compilation process of course).
Maybe somehow you are using gnu-sed on your machine?

Reply:
Ah... very well could be.

Reply:
On my macOS (Catalina), I got:

```
> strings $(which sed) | head
@(#)PROGRAM:sed  PROJECT:text_cmds-101.40.1
$FreeBSD: src/usr.bin/sed/compile.c,v 1.28 2005/08/04 10:05:11 dds Exp $
$FreeBSD: src/usr.bin/sed/main.c,v 1.36 2005/05/10 13:40:50 glebius Exp $
$FreeBSD: src/usr.bin/sed/misc.c,v 1.10 2004/08/09 15:29:41 dds Exp $
$FreeBSD: src/usr.bin/sed/process.c,v 1.39 2005/04/09 14:31:41 stefanf Exp $
```